druid/src/command.rs

```diff
@@ -183,10 +183,18 @@ pub mod sys {
     /// Quit the running application. This command is handled by the druid library.
     pub const QUIT_APP: Selector = Selector::new("druid-builtin.quit-app");
 
-    /// Hide the application. (mac only?)
+    /// Hide the application. (mac only)
+    #[cfg_attr(
```

Comment:
I spent a few minutes thinking about whether this is too fancy, but I think it's a good change. 👍